extension.meta.xml

```diff
@@ -22,8 +22,11 @@
 		<!-- None -->
 	</dependencies>
 	<releases>
+		<release version="1.1" date="2018-04-18" min="2.3.3" max="2.x.x">

```

Comment:
We now use [semver](https://semver.org/) so the version must be `1.1.0` :P

Reply:
Done.